src/modules/endpoint/sagas/__mocks__/loadCities.js

```diff
@@ -11,7 +11,7 @@ function * loadCities (
 ): Saga<Array<CityModel>> {
   const citiesAvailable = yield call(() => dataContainer.citiesAvailable())
 
-  if (!citiesAvailable || forceRefresh) {
+  if (!citiesAvailable) {
```

Comment:
Why was this necessary? this should behave in the same way as the non-mock loadCities.js.

`forceRefresh` should probably not be true in the tests.

Reply:
This is because the tests should not depend on refreshing data and calling the DataContainer.

Reply:
Because it is necessary to also test, that if the lastUpdate is quite a while ago, `loadCityContent` would call loadCities with `forceRefresh===true`.
(Just like `loadCategories` and  `loadEvents` and `loadLanguages` btw.)